config/base.cfg

```diff
@@ -1,6 +1,8 @@
 [buildout]
 extends =
     https://dist.plone.org/release/6.0.15/versions.cfg
+    https://dist.plone.org/release/6.0.15/versions-ecosystem.cfg
+    https://dist.plone.org/release/6.0.15/versions-extra.cfg
```

Comment:
These lines allow us to remove some pinnings (e.g. `ZopeUndo` and `plone.recipe.zeoserver`).